integration_tests/pages/page.ts

```diff
@@ -83,4 +83,30 @@ export default abstract class Page {
   shouldContainBackLink(href: string): void {
     cy.get('.govuk-back-link').should('have.attr', 'href', href)
   }
+
+  checkNavigation(currentPath: string, navigationEnabled: boolean): void {
```

Comment:
Could we call this something like `shouldShow/ContainNavigation` to make the boolean arg line up a bit more clearly with the function name?

Reply:
My thought was that it would be weird to have a parameter asking whether to show navigation in a method called `shouldShowNavigation`, but on reflection I suppose it does make more sense in the method signature when calling it. Does `navigationEnabled` make sense as the parameter though?

Reply:
How would you feel about having two functions here, one doing the `!navigationEnabled` section and one doing the other - i.e. `should` and `shouldNot` show navigation? I don't know if there's a particular benefit to having it as one function when the conditionals are kind of doing different things.